helm/oncall/values.yaml

```diff
@@ -91,13 +91,13 @@ env: []
 ingress:
   enabled: true
 #  className: ""
-  annotations:
-    kubernetes.io/ingress.class: "nginx"
-    cert-manager.io/issuer: "letsencrypt-prod"
-  tls: 
-    - hosts:
-        - "{{ .Values.base_url }}"
-      secretName: certificate-tls
+  annotations: {}
+    # kubernetes.io/ingress.class: "nginx"
+    # cert-manager.io/issuer: "letsencrypt-prod"
+  tls: []
+    # - hosts:
+    #     - "{{ .Values.base_url }}"
+    #   secretName: certificate-tls
```

Comment:
These were disabled since it was not possible to set the `className` with the `kubernetes.io/ingress.class: "nginx"` annotation being present.

Reply:
Removing this lines could break the single click installation, 
how do you think if we add the new ingress definition called `ingress-custom`, which would not have these annotations